odoo/service/server.py

```diff
@@ -412,6 +412,7 @@ def cron_thread(self, number):
                     except Exception:
                         _logger.warning('cron%d encountered an Exception:', number, exc_info=True)
                     thread.start_time = None
+            time.sleep(SLEEP_INTERVAL + number)     # Steve Reich timing style
```

Comment:
gni?

Reply:
I want the cron to be executed as soon as the server is launched, so we don't want to sleep at the start of the cron threads

Reply:
what's `SLEEP_INTERVAL` ? what's `number`?

Reply:
- `SLEEP_INTERVAL` is the minimum interval between the end of the last job and the beginning of the next job
- `number` is the number of the thread, in a range from 0 to `odoo.tools.config['max_cron_threads']`
- `+ number` in `time.sleep(SLEEP_INTERVAL + number)` ensures that not all threads are run at the same time/with the same pattern, and the threads are looking for new jobs in a scattered way, instead of all at the same time.

Each thread is looking for a job to do at each iteration, does it if it found one, and sleep in each case.
What I am doing here is just starting the jobs right after the server started the threads, instead of sleeping first.
I also keep the `time.sleep(number)` so that not all threads run at the exact same time at the start (this part is not absolutely needed)